kuma/javascript/src/payments/pages/terms.jsx

```diff
@@ -0,0 +1,112 @@
+// @flow
+import * as React from 'react';
+import SubHeader from '../components/subheaders/index.jsx';
+
+export const title = 'MDN Web Docs payment terms';
+const TermsPage = () => (
+    <>
+        <SubHeader title={title} />
+        <main
+            className="contributions-page"
+            role="main"
+            data-testid="terms-page"
+        >
+            <section>
+                <h2>Payment Terms</h2>
+
+                <p>Mozilla (that’s us) operates MDN Web Docs.</p>
+                <p>
+                    These Payment Terms cover your payment to MDN.{' '}
+                    <a
+                        href="https://www.mozilla.org/en-US/about/legal/terms/mozilla/"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        The Websites & Communications Terms of Use
+                    </a>{' '}
+                    cover the MDN service itself. If there is any inconsistency
+                    between these terms and the Websites & Communications Terms
+                    of Use, these Payment Terms apply.
```

Comment:
Here, "Payment Terms" really makes sense (capitalizing each word) because that's what's done with "Websites & Communications Terms of Use". 
So, should the document title and `<h1>` *at least* replace `payment terms` with `Payment Terms` and then we can debate, separately, about the phrasing of whether to include the prefix "MDN" or "MDN Web Docs".

Reply:
Technically MDN was rebranded to MDN Web Docs a couple of years ago. Even so, we are still not consistently referring to MDN as MDN Web Docs everywhere. I reckon with all new work we should standardize on always saying MDN Web Docs.